tinyagentos/routes/hub.py

```
@@ -109,3 +147,356 @@ async def put_own_profile(
+        directory = None
+    return {
+        "state": "sent",
+        "target": body.target_fingerprint,
+        "directory": directory,
+    }
+
+
+@router.get("/api/hub/friends/requests")
+async def inbox_friend_requests(
+    request: Request,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)